app/src/main/java/io/intrepid/contest/models/Contest.java

```diff
@@ -0,0 +1,99 @@
+package io.intrepid.contest.models;
+
+import android.os.Parcel;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+import timber.log.Timber;
+
+public class Contest {
+    private UUID contestId;
+    private UUID creatorId;
+    private Date creationDate;
+    private Date lastUpdatedDate;
+    private String title;
+    private String description;
+    private List<Category> categories;
+
+    @Override
+    public String toString() {
+        return title + "/n" +
+                description + "/n" +
+                contestId + "/n" +
+                creatorId + " " + creationDate +
+                " /n size of categories was " + categories.size() +
+                "categories " + categories +
+                " /n Last updated " + lastUpdatedDate;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public static class Builder {
+        public static final int UUID_MIN_LIMIT = 0;
```

Comment:
Anything that we arent currently using should be removed from this list of variables.